src/core/pages/AccountInfo.vue

```diff
@@ -39,88 +59,109 @@
+import set from 'lodash/set';
+import pick from 'lodash/pick';
+import Users from '@api/Users';
 import Input from '@components/form/Input';
 import HtmlModal from '@components/modal/HtmlModal';
-import { AUXILIARY_WITHOUT_COMPANY } from '@data/constants';
+import Modal from '@components/modal/Modal';
+import { NotifyWarning, NotifyPositive, NotifyNegative } from '@components/popup/notify';
+import { frPhoneNumber } from '@helpers/vuelidateCustomVal';
 import { passwordMixin } from '@mixins/passwordMixin';
+import { validationMixin } from 'src/modules/client/mixins/validationMixin';
+import { userMixin } from '@mixins/userMixin';
 import rgpd from 'src/statics/rgpd.html';
 import cguCompani from 'src/statics/cguCompani.html';
 
 export default {
   name: 'AccountInfo',
   metaInfo: { title: 'Mon compte' },
-  mixins: [passwordMixin],
+  mixins: [passwordMixin, validationMixin, userMixin],
   components: {
     'ni-input': Input,
     'ni-html-modal': HtmlModal,
+    'ni-modal': Modal,
   },
   data () {
     return {
-      user: {
+      mergedUserProfile: {
         local: { email: '', password: '' },
-        alenvi: {},
-        contracts: [],
+        contact: { phone: '' },
       },
+      tmpInput: '',
+      emailLock: true,
+      newPasswordModal: false,
       passwordConfirm: '',
+      loading: false,
       rgpd,
       rgpdModal: false,
       cguModal: false,
       cguCompani,
-      tmpInput: '',
       backgroungClass: /\/ad\//.test(this.$router.currentRoute.path) ? 'vendor-background' : 'client-background',
     }
   },
   validations () {
     return {
-      user: {
+      mergedUserProfile: {
         local: {
           email: { required, email },
-          password: { ...this.passwordValidation },
+          password: { required, ...this.passwordValidation },
+        },
+        contact: {
+          phone: { frPhoneNumber },
         },
       },
       passwordConfirm: {
-        required: requiredIf((item) => !!item.user.local.password),
-        sameAsPassword: sameAs((item) => item.user.local.password),
+        required: requiredIf((item) => !!item.mergedUserProfile.local.password),
+        sameAsPassword: sameAs((item) => item.mergedUserProfile.local.password),
       },
     }
   },
-  async mounted () {
+  async created () {
     try {
-      this.user.alenvi = await Users.getById(this.$route.params.id);
-      this.user.local.email = this.user.alenvi.local.email;
+      const user = await Users.getById(this.$route.params.id);
+      this.mergedUserProfile = pick(user, ['local', 'contact']);
     } catch (e) {
       console.error(e);
     }
   },
-  computed: {
-    isAuxiliaryWithoutCompany () {
-      return get(this.user, 'alenvi.client.name', null) === AUXILIARY_WITHOUT_COMPANY;
-    },
-  },
   methods: {
     saveTmp (path) {
-      if (this.tmpInput === '') this.tmpInput = get(this.user, path);
+      if (this.tmpInput === '') this.tmpInput = get(this.mergedUserProfile, path);
```

Comment:
tiens je me demande pourquoi ici on verifie que le tmpInput est '' avant de lui affecter une autre valeur ... 🤔

Reply:
je pense que c'est a cause de la fct emailErrorHandler (dans userMixin) qui revient chercher l'email dans this.tmpInput